src/PHPUnit/HttpMockFacade.php

```diff
@@ -84,4 +90,60 @@ public function each(callable $callback)
     {
         $callback($this);
     }
+
+    /**
+     * Get error output (if exists)
+     *
+     * @return string
+     */
+    public function getErrorOutput()
+    {
+        return static::cleanErrorOutput($this->server->getIncrementalErrorOutput());
+    }
+
+    /**
+     * Get error output (if exists)
+     *
+     * @param string $output
+     *
+     * @return string
+     */
+    public static function cleanErrorOutput($output)
+    {
+        if (!trim($output)) {
+            return '';
+        }
+
+        $errorLines = [];
+
+        foreach (explode(PHP_EOL, $output) as $line) {
+            if(!$line) {
+                continue;
+            }
+            if (!static::endsWith($line, ['Accepted', 'Closing', ' started'])) {
+                //throw new AssertionFailedError($message ?: "PHP Web Server seems to have logged an error: $line");
+                $errorLines[] = $line;
+            }
+        }
+
+        return $errorLines ? implode(PHP_EOL, $errorLines) : '';
+    }
+
+    /**
+     * Returns true if $haystack contains at least one of $needles.
+     *
+     * @param string          $haystack
+     * @param string|string[] $needles
+     *
+     * @return bool
+     */
+    protected static function endsWith($haystack, $needles)
+    {
+        foreach((array)$needles as $needle) {
```

Comment:
💅 space before `(`. What about forcing `$needles` to be an array? 

🎨 Method should be private, no reason to overload it

Reply:
Should be invisble (private) and not static and moved into Server too...
For me it looked like a (perhaps misplaced) utility function so I wasnt so strict. But I will move the array definition 2 lines up it if you like ;-)